test/test_svm_classical.py

```diff
@@ -17,13 +17,12 @@
 
 import numpy as np
 
-from test.common import QiskitAquaTestCase
-
 from qiskit_aqua.utils.multiclass.data_preprocess import *
 from qiskit_aqua.svm_qkernel.data_preprocess import *
 from qiskit_aqua import run_algorithm, get_algorithm_instance
 from qiskit_aqua.input import get_input_instance
 
+from test.common import QiskitAquaTestCase
```

Comment:
test.common is also our local package, it should be categorized with qiskit_aqua together